src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -181,11 +289,32 @@ else if (SourceGroupPacketExtension.SEMANTICS_FID
                     Long primarySSRC = fidEntry.getKey();
                     Long rtxSSRC = fidEntry.getValue();
 
-                    RTPEncodingDesc[] encodings = new RTPEncodingDesc[1];
+                    int numOfTemporal, numOfSpatial;
+                    if (false && ENABLE_SVC)
```

Comment:
I'm lost. If this code is meant to be disabled for the time being, how about setting something like ENABLE_VP9_SUPPORT=false and then using the constant?